src/locales/default.json

```diff
@@ -205,7 +205,7 @@
         },
         "tabs": {
             "claimable": "Claimable now",
-            "currentEstimate": "Current week"
+            "currentEstimate": "Pending"
```

Comment:
Proposing this change as the value presented in this tab will often include estimates from the previous week too (until the orchard is seeded with the roots for the previous week), not just the current one.